test/dlc_tests/conftest.py

```diff
@@ -1241,6 +1242,15 @@ def pytest_runtest_setup(item):
     """
     # Handle quick check tests
     quick_checks_opts = [mark for mark in item.iter_markers(name="quick_checks")]
+
```

Comment:
Just make sure to uncomment this when ready, or remove if not needed

Reply:
done